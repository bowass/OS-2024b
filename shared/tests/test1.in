cal
ls ./todo
history
exit
