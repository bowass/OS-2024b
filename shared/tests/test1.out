     April 2024       
Su Mo Tu We Th Fr Sa  
    1  2  3  4  5  6  
 7  8  9 10 11 12 13  
14 15 16 17 18 19 _2_0  
21 22 23 24 25 26 27  
28 29 30              
                      
a.out
myshell
myshell.c
myshell_f
myshell_f.c
myshell_skeleton.c
submitters.txt
test1.in
test1.out
test2.in
test2.out
test3.in
test3.out
test4.in
test4.out
test5.in
test5.out
test6.in
test6.out
test_temp1.out
my-shell> my-shell> my-shell> 3 	 history
2 	 ls ./todo
1 	 cal
my-shell> 